crates/strategies/blend-strategies/src/liquidation_strategy.rs

```diff
@@ -248,8 +231,20 @@ impl BlendLiquidator {
                         user,
                         pending_fill.target_block.clone()
                     );
-                    self.pending_fill.push(pending_fill);
+                    self.pending_fill.push(pending_fill.clone());
                 }
+
+                let msg = format!(
+                        "<!channel> - Liquidator: {} found a new user liquidation auction for user: {:?} with lot: {:?} bid: {:?} pct_to_fill: {:?} target_block: {:?}",
+                        self.us_public,
+                        user,
+                        auction_data.lot,
+                        auction_data.bid,
+                        pending_fill.pct_to_fill,
+                        pending_fill.target_block - pending_fill.auction_data.block
```

Comment:
will this underflow when target block is 0?

We should consider just logging both `auction_data.block` and `pending_fill.target_block`